source/agora/cli/client/GenTxProcess.d

```diff
@@ -0,0 +1,176 @@
+/*******************************************************************************
+
+    A simple tool which generates valid transactions and sends
+    one transaction periodically to one given node.
+    The interval at which the transaction is sent can be specified.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.client.GenTxProcess;
+
+import agora.api.FullNode;
+import agora.client.Result;
+import agora.common.Amount;
+import agora.common.crypto.Key;
+import agora.common.Hash;
+import agora.common.Task;
+import agora.common.Types;
+import agora.consensus.data.Block;
+import agora.consensus.data.Transaction;
+import agora.consensus.data.genesis.Test;
+import agora.utils.PrettyPrinter;
+import agora.utils.Test;
+
+import core.time;
+import std.algorithm;
+import std.format;
+import std.getopt;
+import std.range;
+import std.stdio;
+
+import vibe.core.core;
+
+public alias APIMaker = API delegate (string address);
+
+/// Option required to generate and send transactions
+private struct GenTxOption
+{
+    /// IP address of node
+    public string host = "0.0.0.0";
+
+    /// Port of node
+    public ushort port = 2826;
+
+    /// The address key to send the output
+    public string address;
+
+    /// Interval of sending transaction
+    public uint interval = 5;
+
+    /// dump output option
+    public bool dump;
+}
+
+/// Parse the command-line arguments of gentx
+public GetoptResult parseGenTxOption (ref GenTxOption op, string[] args)
+{
+    return getopt(
+        args,
+        "ip|i",
+            "IP address of node (default: 0.0.0.0)",
+            &op.host,
+
+        "port|p",
+            "Port of node (default: 2826)",
+            &op.port,
+
+        "dest|d",
+            "The address key to send the output",
+            &op.address,
+
+        "interval|t",
+            "Interval of sending transaction",
+            &op.interval,
```

Comment:
```suggestion
            (string value ) { op.interval = value.to!int.seconds; },
```

This assumes that the interval given at command-line always specifies seconds, and not milliseconds for example.